imagemagick required for 2FA (#4559)
owncloud/docs: modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc

To provide 2FA functionality, an app like the {oc-marketplace-url}/apps/twofactor_totp[2-Factor Authentication] needs to be installed and enabled.

If a two-factor provider app is enabled, it is enabled for all users by default *but* _a user has to opt in_, though the provider can decide whether or not the user has to pass the challenge.

> TIP: For the opt-in function to work, ImageMagick needs to be installed to create the QR code. Depending on your operating system, this package may be installed by default or you'll have to manually take care of that.

Author: ```suggestion
TIP: For the opt-in function to work, ImageMagick and php-imagick are required to create the QR code. Depending on your operating system, these packages may need to be installed manually.
```
